addons/mail/static/src/core/common/message.xml

```diff
@@ -2,10 +2,13 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.Message">
-        <div t-if="message.isNotification" class="o-mail-NotificationMessage text-break mx-auto text-muted opacity-75 px-3 text-center smaller" t-on-click="onClickNotificationMessage"  t-att-class="props.className" t-ref="root">
-            <i t-if="message.notificationIcon" t-attf-class="{{ message.notificationIcon }} me-1"/>
-            <span class="o-mail-NotificationMessage-author d-inline" t-if="message.author and !message.body.includes(escape(message.author.name))" t-esc="message.author.name"/> <t t-out="message.body"/>
-        </div>
+        <t t-if="message.isNotification">
+            <CallNotificationMessage t-if="message.notificationType === 'call'" t-props="callNotificationProps" />
+            <div t-else="" class="o-mail-NotificationMessage text-break mx-auto text-muted opacity-75 px-3 text-center smaller" t-on-click="onClickNotificationMessage"  t-att-class="props.className" t-ref="root">
+                <i t-if="message.notificationIcon" t-attf-class="{{ message.notificationIcon }} me-1"/>
+                <span class="o-mail-NotificationMessage-author d-inline" t-if="message.author and !message.body.includes(escape(message.author.name))" t-esc="message.author.name"/> <t t-out="message.body"/>
+            </div>
+        </t>
```

Comment:
I suggest to make a component / template for NotificationMessage, and have `CallNotificationMessage` reuse that component template for style.

NotificationMessage should be small and have muted visual, as to be subtle in comparison to actual messages in conversation. The current visual is too distracting and big:

<img width="383" alt="Screenshot 2024-12-12 at 17 56 56" src="https://github.com/user-attachments/assets/ce85862c-38ee-4fda-ac63-2965506f262d" />

Style is bad when call ended:

<img width="383" alt="Screenshot 2024-12-12 at 17 57 35" src="https://github.com/user-attachments/assets/18b453cf-d448-4879-ab77-9d5bc7af979b" />

By reusing `NotificationMessage` style as template and forcing being brief in notification text content, this should look like this:

<img width="383" alt="Screenshot 2024-12-12 at 18 04 34" src="https://github.com/user-attachments/assets/db81c445-efd9-4d51-8784-0d0721e45871" />

Ignore the "Today" that's repeated 3 times: I plan to make improvements in master at some point to have date section being sticky and thus the datetime of message and notification can have the dateday omitted.


So by reusing the `NotificationMessage` template / component, you will reuse style and `message.notificationIcon`. For the date, this can be an option that is either explicit in notification payload or is deduced by notification type.